Query.php

```diff
@@ -164,7 +165,9 @@ public function andFilterCompare($name, $value, $defaultOperator = '=')
         } else {
             $op = $defaultOperator;
         }
-
+        if(isset($result['$err'])){
```

Comment:
This is definetely wrong: there is no variable `$result` inside `andFilterCompare` method.